tests/e2e/test_post_transaction.py

```diff
@@ -26,9 +26,9 @@ def test_token_decimals(self):
         token_transfer = Transfer(
             token=self.cow,
             amount_wei=15,
-            receiver=self.receiver,
+            _solver=self.receiver,
         )
-        native_transfer = Transfer(token=None, receiver=self.receiver, amount_wei=2)
+        native_transfer = Transfer(token=None, _solver=self.receiver, amount_wei=2)
```

Comment:
Only a test, but here I'm confuse as well that we set the solver field to a potential redirect target.

Reply:
You are correct to point this out and this was not at all intended... just missed during the name change. The linter didn't catch this because we don't lint the tests. Similar for a few line below.

Fixed.